services/backend/services/v3/config/config.local.js

```diff
@@ -37,5 +37,6 @@ module.exports = {
   queue: process.env.ENABLE_JOBS? "rabbitmq": undefined,
   logbook: {
     enabled: false
-  }
+  },
+  expressSessionSecret: "someSecret",
```

Comment:
should this be set only if OIDC is selected?

Reply:
I think that it is indeed only used for OIDC, do you think we can merge JS files using jq?

Reply:
tbh, I would simply add the conditional logic to the JS file. It's probably not the most elegant way to do, but it will avoid extra complexity. Something like [this](https://github.com/SciCatProject/scicatlive/blob/main/services/backend/services/v3/config/config.local.js#L37) I think will work